plugin-scalebar/src/main/java/com/mapbox/pluginscalebar/ScaleBarWidget.java

```diff
@@ -363,6 +364,22 @@ private String getDistanceText(int distance) {
     }
   }
 
+  /**
+   * Set the ratio of scale bar max width compared with MapView width.
+   * @param ratio the ratio scale bar will use.
+   */
+  public void setRatio(float ratio) {
```

Comment:
Maybe rename to `setMaxWidthRatio` , ratio itself would mean the screen pixel ratio